.evergreen/csfle/activate_venv.sh

```diff
@@ -22,4 +22,5 @@ fi
 # install dependencies on first run
 if [ ! -z $FIRST_RUN ]; then
   pip install --upgrade boto3
+  pip install --upgrade pykmip
```

Comment:
Let's pin these dependencies down a bit so that that new releases won't break our tests. boto3 is stable whereas pykmip is not so we can do:
```
pip install --upgrade boto3~=1.19 pykmip~=0.10.0
```

Reply:
Good idea. Done.